src/Dropdown/Dropdown.js

```diff
@@ -47,6 +47,25 @@ class Dropdown extends Util.mixin(BindMixin, KeyDownMixin) {
     }
   }
 
+  componentDidMount() {
+    if (typeof this.props.scrollContainer === 'string') {
+      this.container = DOMUtil.closest(ReactDOM.findDOMNode(this),
+        this.props.scrollContainer) || window;
+
+      if (this.container.parentElement != null
+        && this.props.scrollContainerParentSelector != null
+        && this.container.parentElement[DOMUtil.matchesFn](
```

Comment:
Will it always be the immediate parent? Or should we walk up the tree till we find a match?